src/coreclr/System.Private.CoreLib/src/System/Variant.cs

```diff
@@ -328,7 +328,7 @@ CVType switch
                 CV_U4 => (uint)_data,
                 CV_I8 => _data,
                 CV_U8 => (ulong)_data,
-                CV_R4 => BitConverter.Int32BitsToSingle((int)_data),
+                CV_R4 => BitConverter.UInt32BitsToSingle((uint)_data),
```

Comment:
Why change this one?

Reply:
Changed it so that both places in Variant use the same pair of APIs, the unsigned ones.